faqs.rst

```diff
@@ -205,6 +205,37 @@ Caveats
 The debugger exposes entire gauge-js runner code.
 You need to be quick enough to hit continue in the browser when node-inspector launches. If this takes too long, gauge will timeout connecting to the API. A workaround for this is to increase the runner_connection_timeout property to an acceptable value.
 
+
+gRPC issues
+^^^^^^^^^^^^
+
+gauge-js uses `gRPC <https://github.com/grpc/grpc-node/tree/master/packages/grpc-native-core>`_ to communicate with Gauge.
+The package  ``gRPC`` is a native module, and uses native bindings.
+
+When the nodejs is updated the ``gauge run`` fails because the installed binaries are not compatible with the version.
```

Comment:
When Node is updated, ``gauge run`` fails due to binary incompatibility with the newer version.